package com.rcore.domain.auth.credential.exceptions;

public class CredentialWithPhoneAlreadyExistException extends Exception {
}
